app/src/main/java/protect/card_locker/AboutActivity.java

```diff
@@ -166,7 +167,11 @@ public void onClick(View view) {
 
         Intent intent = new Intent(Intent.ACTION_VIEW);
         intent.setData(Uri.parse(url));
-        startActivity(intent);
+        if (intent.resolveActivity(getPackageManager()) != null) {
```

Comment:
Please instead catch the exception when startActivity failed. API level 30 added extra restrictions to resolveActivity which may make it return null even if a browser is available: https://stackoverflow.com/questions/62535856/intent-resolveactivity-returns-null-in-api-30

There's an example in https://github.com/CatimaLoyalty/Android/blob/0189f13ee652bd154f1171981880d2bac4f63e9d/app/src/main/java/protect/card_locker/ImportExportActivity.java#L148-L153.

Reply:
Oops, I wasn't aware of the new restrictions. I updated the PR, please have another look.